PROTOCOL.md

```diff
@@ -400,6 +418,11 @@ animation) and `|[silent]` (suppress message).
 > The Pokémon `POKEMON` has transformed into `SPECIES` by the effect of Transform 
 > or the ability Imposter.
 
+`|-mega|POKEMON|MEGASTONE`
+
+> The `POKEMON` used `MEGASTONE` to Mega Evolve. If a Pokémon have several Mega Evolutions,
+> `MEGASTONE` specifies which one it changed into.
```

Comment:
Now that I think about it, you don't need to mention things about Pokemon having several Mega Evolutions, since you already mentioned that it used a particular Mega Stone to Mega Evolve.